cfpb/consumerfinance.gov
# Redirects to correct ireg 404s

We have a series of 404 errors triggered in our Regulations section (https://www.consumerfinance.gov/rules-policy/regulations/) caused by how relative URL paths are being created on certain pages. This is a known tech issue that should be solved in code. However, in the meantime, this PR manually sets up redirects for these 404s (based on use of our internal content crawler's 404 report) so that website functionality is not hamstrung further by this work's placement in the backlog.

When the technological issue is resolved, these redirects should be able to be removed safely.

---




## Additions

- 75 new redirects all focused on a systemic issue in the generation of links in the regulations section


## How to test this PR

1. localhost
2. Navigate to /rules-policy/regulations/1003/2018-01-01/a/
3. Find the link to /rules-policy/regulations/1003/1002/9/
4. Confirm that the link properly resolves to /rules-policy/regulations/1002/9/
5. Check that all other redirects resolve to non-404s


## Notes and todos

- As mentioned, these should be removed if/when the systemic issue that causes them is resolved; likewise, more may need to be added if other links of this nature are generated in the future, until the larger issue is resolved.


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
